README.md

```diff
@@ -49,6 +49,33 @@ This list is just for Red Hat contributions to projects hosted on GitHub.  See h
 ### Hosting projects on /RedHatOfficial
 
 Open an issue in *this* repo, and the owners of the RedHatOfficial organization will get in touch with you.
+
+## Quick start
+    ```
```

Comment:
This code block isn't upgrading correctly - can you tidy up the spacing a bit?

Reply:
I have not understood the line 49,51 could you pls explain it little bit?